src/components/Header/BaseHeaderNavigation.jsx

```diff
@@ -65,14 +67,27 @@ const BaseHeaderNavigation = ({ children, testMode = false, ...attrs }) => {
     }
   };
 
+  // Avatar and Search components need to be rendered inside the nav tag, but outside of the list
+  const listItems = [];
+  const otherHeaderMenuItems = [];
+  React.Children.forEach(children, (child) => {
+    if ([Header.Avatar, Header.Search].includes(child.type)) {
```

Comment:
I think it may make more sense to reverse the sense of this test. That is, if the child is a `BaseHeaderMenuItem` add it to the `listItems` and otherwise it goes outside the list. Since we know `BaseHeaderMenuItem` is wrapped in a `li`, it can go in the list and any additional content is outside the list content.

Reply:
Works for me. Done. Thanks.